### PR TITLE
chore(demo-playwright): test for input-month

### PR DESCRIPTION
**Pull request by VidhiRambhia**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [x] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Addresses https://github.com/taiga-family/taiga-ui/issues/4912

## What is the new behaviour?

This PR adds an e2e test (with playwright) for input-month

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        let documentationPage: TuiDocumentationPagePO;
+        let example: Locator;
+        let inputMonth: TuiInputMonthPO;
+
+        test.use({
+            viewport: {width: 360, height: 500},
+        });
+
+        beforeEach(async ({page}) => {
+            await tuiGoto(page, `components/input-month/`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:17]: Probably we can drop this line

(Locators can initialized even before navigation)

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            documentationPage = new TuiDocumentationPagePO(page);
+            example = documentationPage.apiPageExample;
+            inputMonth = new TuiInputMonthPO(example.locator(`tui-input-month`));
+        });
+
+        test(`Maximum month less than current month`, async ({page}) => {
+            await tuiGoto(page, `components/input-month/API?tuiMode=null&max$=1`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:24]:
```suggestion
            await tuiGoto(page, `components/input-month/API?max$=1`);
```

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            await inputMonth.textfield.click();
+
+            await documentationPage.prepareApiPageBeforeScreenshot();
+            await expect(page).toHaveScreenshot(`input-month-maximum-month.png`);
+        });
+
+        test(`Minimum month more than current month`, async ({page}) => {
+            await tuiGoto(page, `components/input-month/API?tuiMode=null&min$=3`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:32]:
```suggestion
            await tuiGoto(page, `components/input-month/API?min$=3`);
```

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            example = documentationPage.apiPageExample;
+            inputMonth = new TuiInputMonthPO(example.locator(`tui-input-month`));
+        });
+
+        test(`Maximum month less than current month`, async ({page}) => {
+            await tuiGoto(page, `components/input-month/API?tuiMode=null&max$=1`);
+            await inputMonth.textfield.click();
+
+            await documentationPage.prepareApiPageBeforeScreenshot();
+            await expect(page).toHaveScreenshot(`input-month-maximum-month.png`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:28]: ✅ 
input-month-maximum-month-chromium-darwin

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        });
+
+        test(`Minimum month more than current month`, async ({page}) => {
+            await tuiGoto(page, `components/input-month/API?tuiMode=null&min$=3`);
+            await inputMonth.textfield.click();
+
+            await documentationPage.prepareApiPageBeforeScreenshot();
+            await expect(page).toHaveScreenshot(`input-month-minimum-month.png`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:36]: ✅ 
input-month-minimum-month-chromium-darwin

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+const {describe, beforeEach} = test;
+
+describe(`InputMonth`, () => {
+    describe(`API`, () => {
+        let documentationPage: TuiDocumentationPagePO;
+        let example: Locator;
+        let inputMonth: TuiInputMonthPO;
+
+        test.use({
+            viewport: {width: 360, height: 500},
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:13]:
```suggestion
            viewport: {width: 360, height: 410},
```

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts (hunk 4 above)
Comment (nsbarsukov) [reply projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:28]: (but we can decrease the height a little bit)

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts (hunk 5 above)
Comment (nsbarsukov) [reply projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:36]: (but we can decrease the height a little bit)

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:17]:
```suggestion
```

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo-playwright/tests/kit/input-month/input-month.spec.ts [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    describe(`API`, () => {
+        let documentationPage: TuiDocumentationPagePO;
+        let example: Locator;
+        let inputMonth: TuiInputMonthPO;
+
+        test.use({
+            viewport: {width: 360, height: 410},
+        });
+
+        beforeEach(async ({page}) => {
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month/input-month.spec.ts:16]:
```suggestion
        beforeEach(({page}) => {
```

**Pull request merged**